spec/models/bundle_context_spec.rb

```diff
@@ -79,7 +79,7 @@
 
   describe "#staging_dir" do
     it 'is hardcoded to the correct path' do
-      expect(described_class.new.staging_dir).to eq '/dor/assembly'
+      expect(described_class.new.staging_dir).to eq '/tmp/assembly'
```

Comment:
this goes with comment on previous commit re: this being a setting.  Maybe not needed in bundle_context at all -- it will be same setting for ALL bundles.  Maybe there are other candidates for settings where it is the same for all bundles?

Also - should this stay as a separate commit, in a perfect world, or be squashed with first?  (don't care so much right now but for future practice -- review would be easier)

Reply:
fine with follow on ticket for this.

Reply:
#224